stakingModel/contracts/SimpleDAIStaking.sol

```diff
@@ -58,22 +59,26 @@ contract SimpleDAIStaking is DSMath, Pausable, SchemeGuard {
     //100% for phase0 POC
     uint32 public avgInterestDonatedRatio = 1e6;
 
+    address fundManager;
+
     modifier onlyFundManager {
-        require(msg.sender == owner(), "Only FundManager can call this method");
+        require(msg.sender == fundManager, "Only FundManager can call this method");
         _;
     }
 
     constructor(
         address _dai,
         address _cDai,
         address _fundManager,
-        uint256 _blockInterval
-    ) public SchemeGuard(Avatar(address(0))) {
+        uint256 _blockInterval,
+        Avatar _avatar,
+        Identity _identity
+    ) public FeelessScheme(_identity, _avatar) {
         dai = ERC20(_dai);
         cDai = cERC20(_cDai);
         blockInterval = _blockInterval;
         lastUBICollection = block.number;
-        transferOwnership(_fundManager);
+        fundManager = _fundManager;
     }
```

Comment:
we should add the avatar as a pauser and renounce the owner
addPauser
renouncePauser

Reply:
its fine like that